app/build.gradle

```diff
@@ -71,7 +71,9 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:18.0.0'
     implementation "androidx.fragment:fragment:1.3.2"
     implementation 'androidx.room:room-runtime:2.2.6'
-    implementation 'com.google.firebase:firebase-database:18.0.1' //implementation 'com.google.firebase:firebase-database:19.5.1' //implementation 'com.google.firebase:firebase-database:18.0.1'
+
+    implementation 'com.google.firebase:firebase-database:19.7.0'
```

Comment:
Yes the latest version !

Reply:
Yes, our Firebase Runloop error will soon be gone!